SPECS/kernel/kernel.spec

```diff
@@ -2,7 +2,7 @@
 Summary:        Linux Kernel
 Name:           kernel
 Version:        5.4.51
-Release:        5%{?dist}
+Release:        6%{?dist}
```

Comment:
Please also bump the Release number and add a changelog entry in `SPECS-SIGNED/kernel-signed-x64` and `SPECS-SIGNED/kernel-signed-aarch64`

Reply:
Thank you. I will do the needful.

Reply:
Added